my-app/src/services/employees.service.js

```diff
@@ -0,0 +1,39 @@
+import Constants from "../constants/constants.service";
+import { getToken } from "./auth.service";
+import { Component } from "react";
+
+const baseUrl = Constants.getBaseUrl();
+const options = {
+  method: "GET",
+  headers: {
+    authorization: getToken(),
+  },
+};
+
+export default class EmployeesService extends Component {
```

Comment:
The same here

Reply:
okaaay,, I think I got it) when I'm not rendering anything I dnt need to use extendsComponent)